src/Popover.js

```diff
@@ -115,18 +113,15 @@ Popover.defaultProps = {
  */
 Popover.propTypes = {
     children: propTypes.node.isRequired,
-    reference: propTypes.oneOfType([
-        elementRefPropType,
-        propTypes.instanceOf(Element),
-    ]).isRequired,
```

Comment:
Should the reference no longer be required?

Reply:
> Should the reference no longer be required? 

No, for the following reasons:
- Most importantly, because in the new implementation using `usePopper` from `react-popper`, this doesn't cause a runtime error anymore.
- But also because we are now also allowing DOM nodes directly, and it's a fairly likely scenario that these are undefined at some point in the render cycle

> By the way (sidenote), weren't we moving away from common prop-types?

I don't really know about that... In `@dhis2/ui-core` they are still there but in a separate package. In any case, the ones we had needed to be corrected/updated.